convert/src/vertex_to_bd/badgerdoc_to_vertex_use_case.py

```diff
@@ -0,0 +1,84 @@
+import tempfile
+from pathlib import Path
+from typing import NamedTuple
+
+from src.vertex_to_bd.labelstudio_format import LabelStudioFormat
+
+from .models import S3Path
+from .models.bd_annotation_model import BadgerdocAnnotation
+from .models.bd_tokens_model import Page
+
+
+class BadgerdocData(NamedTuple):
+    page: Page
+    annotation: BadgerdocAnnotation
+
+
+class BDToVertexConvertUseCase:
```

Comment:
May you make class name more "elegant"?

Reply:
It seems like this class works with s3 - fetch, upload, etc from s3. We actually don't need class at all. 
`download_badgerdoc_from_s3` returns not the object of this class, but `BadgerdocData`.

Reply:
It is a UseCase: https://www.geeksforgeeks.org/designing-use-cases-for-a-project/
As we need to deploy this code as soon as possible, I offer to discuss UseCase on a meeting. And create an issue about refactoring if it is needed.